primitives/core/src/ecdsa.rs

```diff
@@ -499,7 +499,7 @@ impl TraitPair for Pair {
 impl Pair {
 	/// Get the seed for this key.
 	pub fn seed(&self) -> Seed {
-		self.secret.serialize_secret()
+		self.secret.secret_bytes()
```

Comment:
👀

Reply:
it's a breaking change of secp256k1 api

Reply:
Just a rename indeed [commit](https://github.com/rust-bitcoin/rust-secp256k1/commit/5c7c76eb7494d3a031301d40d1ccebd0f9b9d7ae).